jschon/vocabulary/core.py

```diff
@@ -62,14 +63,15 @@ def __init__(self, parentschema: JSONSchema, value: Mapping[str, bool]):
                     raise JSONSchemaError(f"The metaschema requires an unrecognized vocabulary '{vocab_uri}'")
 
 
-class IdKeyword(Keyword):
+class IdKeyword_2019_09(Keyword):
```

Comment:
This should remain `IdKeyword`, as it's still the "$id" implementation for the latest published version of the spec.

The convention I'm following (which I'll document at some point) is to consign a keyword implementation to the `legacy` module as soon as it diverges from the latest published spec, suffixed with the last version in which it is valid. So, once draft-next is published, this "$id" implementation will be moved to `legacy` and renamed `IdKeyword_2020_12`, while `IdKeyword_Next` will be moved from `future` to `core` and renamed to `IdKeyword`.

Reply:
Oh, good to know, I'll update the `contains` thing for draft-next which is wrong in the same way.